### PR TITLE
Updated readme for open source

### PR DESCRIPTION
**Pull request by elspeth-rabid**
A few extra comments around contributing and less talk of Rabid developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,32 @@
 
 This repository holds the code necessary to develop the NZSL Dictionary App for Android. This codebase can be imported into Android Studio and you are invited to fork it to add features and bug fixes.
 
-The original code was developed by [Greg Hewgill](http://hewgill.com/) and generously gifted to the [Deaf Studies Research Unit](http://www.victoria.ac.nz/lals/centres-and-institutes/dsru) of [Victoria University of Wellignton](http://www.victoria.ac.nz/). It is maintained by [Rabid Technologies](http://www.rabid.co.nz/). 
+The original code was developed by [Greg Hewgill](http://hewgill.com/) and generously gifted to the [Deaf Studies Research Unit](http://www.victoria.ac.nz/lals/centres-and-institutes/dsru) of [Victoria University of Wellignton](http://www.victoria.ac.nz/). It is maintained by [Ackama](http://www.ackama.com/).
```
Comment (Rabid-Dan) [inline README.md]: 🐘 should the Ackama link be https?

File: README.md [hunk 2]
```diff
@@ -2,20 +2,32 @@
...
+# Contributions
+
+Contributions are welcome for this project. Please comment on an issue if you are willing to work on it and an administrator will give you further information if required. 
+To contribute, make a fork of this repository and branch off the master branch.
+Create a pull request against the ODNZSL master branch.
+Two approvals are required before a PR can be merged.
+
+# Deployment
+
+To deploy to production: merge a feature branch into ODNZSL master.
```
Comment (ghewgill) [inline README.md]: Spelling (production).

File: README.md [hunk 3]
```diff
@@ -2,20 +2,32 @@
...
+A repositiory adminstrator will perform the necessary actions when a release is planned.
```
Comment (ghewgill) [inline README.md]: Spelling (repository, administrator).

File: README.md (hunk 1 above)
Comment (elspeth-rabid) [reply README.md]: Yes!

File: README.md (hunk 2 above)
Comment (elspeth-rabid) [reply README.md]: Hi Greg! I can't see the typo? 👀

File: README.md (hunk 3 above)
Comment (elspeth-rabid) [reply README.md]: Oh wow 😄

**Pull request merged**